### ISSUE TITLE
"Resume" should respect the current zoom level.

### ISSUE THREAD

**Issue by rationalcoder** [wolfpld/tracy#69]
When using the viewer, the live updating should respect the current zoom level. It's okay for "Resume" to start you at the tens of milliseconds range, but once you zoom out it should respect your zoom level. Otherwise, you have to keep manually dragging the view to the latest events, and the most zoomed in level is pretty useless for certain types of applications.

**Comment by wolfpld** [comment]
You might want to try https://github.com/wolfpld/tracy/pull/63.

**Comment by rationalcoder** [comment]
I have since worked around this by introducing a bogus concept of frames in our non-frame-based applications, and it seems like the zoom level is determined by frame times, so I have achieved reasonable behavior. Also, I'm not sure #63 helps me.

A separate feature request would be support for non-frame-based applications (io/interrupt/event driven ones), in which case respecting the zoom level is more important. In our applications, we tend to want to watch the profile output at some convenient zoom level, make a call (generate an event in the system that we want to profile), then pause and analyze what happened.

Minimal work involved would be 1) respecting the zoom level, and 2) not have one giant frame extending to infinity.
